### PR TITLE
AWS Config and Security Hub module

### PR DESCRIPTION
**Pull request by VolkerBohne**
and change in pre-commit hook to work with Jetbrains IDEs (f9f699e).

### CODE REVIEW

File: modules/config/main.tf [hunk 1]
```diff
@@ -0,0 +1,165 @@
+##
+## AWS Config
+
+resource "aws_iam_role" "config" {
+  name = var.config_name
+
+  assume_role_policy = <<POLICY
```
Comment (michaelkrieg) [inline modules/config/main.tf]: can we have this as a policy document?

File: modules/config/main.tf [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  ignore_public_acls      = true
+  restrict_public_buckets = true
+  depends_on              = [time_sleep.wait_for_bucket_created]
+}
+
+resource "aws_s3_bucket_policy" "config_bucket_policy" {
+  bucket     = aws_s3_bucket.config.id
+  depends_on = [time_sleep.wait_for_bucket_created]
+
+  policy = <<POLICY
```
Comment (michaelkrieg) [inline modules/config/main.tf]: can we have this as a policy document?

**Review by Piethan — APPROVED** [review]
Looks good

**Review by michaelkrieg — APPROVED** [review]

File: modules/config/main.tf (hunk 1 above)
Comment (VolkerBohne) [reply modules/config/main.tf]: done and fixed.

File: modules/config/main.tf (hunk 2 above)
Comment (VolkerBohne) [reply modules/config/main.tf]: done and fixed.

**Pull request merged**